scripts/api-compatibility-check/breaking-api-check.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+print_usage() {
+        cat <<HELP_USAGE
+Usage:
+        $0 [-p MapboxMaps.zip]
+    -p  Path to pre-packaged MapboxMaps
+HELP_USAGE
+}
+
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+PACKAGER_DIR="$SCRIPT_DIR/../release/packager"
+
+API_DIGESTER_PATH="$(xcode-select -p)/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-api-digester"
+
+ARTIFACTS_ZIP_PATH=""
+
+main() {
+    local products_dir="$SCRIPT_DIR"
+    local product_artifacts_dir="$products_dir/artifacts"
+
+    rm -rf "$SCRIPT_DIR/API"
+    rm -rf "$product_artifacts_dir"
+
+    if [[ ! -f "$ARTIFACTS_ZIP_PATH" ]]; then
+        cd "$PACKAGER_DIR" || exit 1
+        ./package-mapbox-maps.sh
+
+        ARTIFACTS_ZIP_PATH="$PACKAGER_DIR/MapboxMaps.zip"
+    fi
```

Comment:
Would you want to go back to the original working directory after this is complete?

Reply:
Nope as all steps are path-indepedent in the script. However, I have to switch folder for the path-dependent scripts or `zip` command.